docs/pages/zkvm/sdk-benchmarking.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "Nexus zkVM SDK Benchmark"
+lang: en-US
+description: "Benchmark your Nexus project using the SDK"
+image: "/nexus-head.png"
+---
+
+import Image from "next/image";
+
+import { Steps } from "nextra/components";
+
+## Reproducible Proving and Verification
+
+<Steps>
+
+This example demonstrates how to use the Nexus SDK to generate proving and verification keys, compile a guest program, prove its execution, and verify the proof. The example uses the `guest` package as an example guest program.
```

Comment:
This bit should come before the steps start.

Reply:
> The example uses the `guest` package as an example guest program.

This sentence comes across as confusing and tautological. Maybe rename the package from `guest` to `example` or something?

Reply:
I replaced the entire paragraph with new one simpler: 

```
This benchmark demonstrates the performance of Nexus zkVM using a Fibonacci sequence calculation as an example. We'll analyze both the guest program's RISC-V cycle count and the host program's execution time for various Fibonacci sequence lengths.
```

Updated in [36f79a4](https://github.com/nexus-xyz/nexus-
zkvm/pull/271/commits/36f79a4ffc509fe44f4c288746c674652d698a4b)